playbooks/roles/update-completion-aggregator/templates/update_completion_aggregator_script.j2

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Ensure only one instance of the script is running
+LOCKFILE=/edx/var/completion-aggregator.lock
+if [ -e ${LOCKFILE} ] && kill -0 `cat ${LOCKFILE}`; then
+    echo "Completion aggregator update already running!"
+    exit
+fi
+# make sure the lockfile is removed when we exit and then claim it
+trap "rm -f ${LOCKFILE}; exit" INT TERM EXIT
+echo $$ > ${LOCKFILE}
+
+# Set environment
+. /edx/app/edxapp/edxapp_env
+
+# Source virtualenv
+source /edx/app/edxapp/venvs/edxapp/bin/activate
```

Comment:
not a problem, but `edxapp_env` sets the desired python executable too, so it is kind of a duplicate, but safer to do this way.